Jenkinsfile

```diff
@@ -93,3 +105,10 @@ def buildTestAndScanImage(name) {
   scanAndReport("${name}:${TAG}", "HIGH", false)
   scanAndReport("${name}:${TAG}", "NONE", true)
 }
+
+def buildTestAndScanImageUBI(name) {
+  sh "./${name}/build.sh ${TAG}"
+  //sh "./${name}/test.sh ${TAG}"
```

Comment:
did you try to use the ubuntu or phusion test scripts out of the box and see how it looked for this image?

Reply:
The test scripts searches for specific outputs for openssl commands and ubuntu (apt) specific commands so it will break.

Reply:
In my last commit I added a test and reverted back to original buildTestAndScanImage :)